### ISSUE TITLE
Dutch badware domains flooding Google search results

### ISSUE THREAD

**Issue by randomtroller** [uBlockOrigin/uAssets#9105]
### Prerequisites

- [x] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- [x] I tried to reproduce the issue when...
    - [x] uBlock Origin is the only extension
    - [x] uBlock Origin with default lists/settings
    - [ ] using a new, unmodified browser profile
- [x] I am running the latest version of uBlock Origin

### URL(s) where the issue occurs
bonbonijezero.xyz
sladicagora.xyz
kruhzgodovina.xyz
mandljigeografija.xyz
zajtrkhisa.xyz
kuminaluna.xyz
piscanecmatematika.xyz
hranavrt.xyz
maslootok.xyz
clublwd.nl
samakita.nl
cmbcleaningservices.nl
radiobitchesawards.nl
bodeweshuisarts.nl
acneventbuilding.nl
bemini.nl
dpmadives.nl
hardcoreteam.nl
platformdekrijger.nl
jiponline.nl
ronhaleber.nl
deutscherklub.nl
keeponhosting.nl
decomschehoeve.nl
kcdn.nl
facebookgroepen.nl
provisible.nl
buitenplaatsgroenoord.nl
zoo-and-me.nl
fietstassen-en-fietsmanden.nl
cocaine-test.nl
databankkinderopvang.nl
thenailfactory.nl
saba-hotels.nl
genseo.nl
webmaak.nl
southernomelet.nl
gretch.nl
pensiontamboer.nl
fiscathema.nl
archicentrumhoorn.nl
bfree4work.nl
atelierbuiten.nl
ademvangst.nl
visibleuniverse.nl
josefwillems.nl
anastasiafotografie.nl
uva.nl
volledigonline.nl
vermeerenmeer.nl
zijlstramakelaardij.nl
leningondernemers.nl
dezwarejagendejongens.nl
kattenhulpverlening.nl
xbmod.nl
lagiara.nl
gadgets4you.nl
cameraverhuurxl.nl
magic-games.nl
hetatkinsforum.nl
tatooforums.nl
myfashioncloset.nl
studeersnel.nl
issuu.nl
popcornenpumps.nl
hairmoso.nl
exploration-bb.nl
gitar.nl
docplayer.nl
sailorsforsailors.nl
cpa-inc.nl
bloggingyou.nl
4x4unlimited.nl
nieuwelitairelan.nl
heiwerken-pvantwout.nl
testje2.nl
psv35.nl
safekidszone.nl
admvanbeers.nl
store4pets.nl
duoprojectinrichting.nl
arbefis.nl
attentum.nl
aandeduinvoet.nl
saba-trails.nl
saamvastgoed.nl
16h.nl
bekledingexpress.nl
catterykeroka.nl
basdistributie.nl
mindillusion.nl
cdageldrop-mierlo.nl
meubel2outlet.nl
onyougo.nl
vvvolst.nl
rickid.nl
propperbox.nl
tatooforums.nl

### Describe the issue
A Google search for `site:(domain)` shows that these websites have from 10k to up to a million search results that all redirect to scam pages such as "prizes". They flood search results and make them look like legit search results in Slovenia and Turkey.

 

### Versions

- Browser/version: Firefox 88.0.1 (64-bit)
- uBlock Origin version: v1.35.2

### Settings

- Advanced mode, JavaScript disabled, all available filters enabled.

### Notes
By looking in archive.org, I found out that most of these websites were once legit websites that either got infected or sold. I've also emailed to abuse email of the domain registrar but got no response.

**Comment by krystian3w** [comment]
https://github.com/uBlockOrigin/uAssets/issues/9124

**Comment by krystian3w** [comment]
Any support Polish translation as native (no by mechanism embeded in Google Chrome)?

**Comment by krystian3w** [comment]
@BPower0036 maybe worth block for ABP/AdGuard users.

**Comment by krystian3w** [comment]
@Yuki2718 regex's maybe need new update to cover new links like:

`logar65dty.live/kuucbxlp/?cid=`

**Comment by BPower0036** [comment]
> @BPower0036 maybe worth block for ABP/AdGuard users.

My filterlist is made only for uBlock Origin. And because these pages are already blocked by the default uBlock filters. I'm not going to add it to my Dutch list. My Dutch list is only a **supplement** for the already default uBlock Origin Filters.

**Comment by Yuki2718** [comment]
Updated regex to catch them: https://github.com/uBlockOrigin/uAssets/commit/9543db7dd7728c5b93b570debcf95d1fc1d5290d

**Comment by Kuijkens** [comment]
Why is studeersnel.nl on this list?

I'm the co-founder of https://www.studeersnel.nl, which is actually the Dutch version of studocu.com, and it is definitely legit. 
We are not infected, it's not malware, it's not spam and it's definitely not redirecting to Slovenia or Turkish websites.

This note is a false claim:
>  I've also emailed to abuse email of the domain registrar but got no response.

I have never received such email.

Could you please remove my domain since it's now blocking my users, who also use uBlock, and completely breaking the website.

Note:
This list seems completely random and there are a lot of very genuine and legitimate websites on it. I would recommend removing this list and reviewing all items one by one.

**Comment by mapx-** [comment]
https://github.com/uBlockOrigin/uAssets/commit/6572fcebf15d94564eb66cee0e25c6d8847c9ac8

**Comment by krystian3w** [comment]
> This note is a false claim:
>
>>    I've also emailed to abuse email of the domain registrar but got no response.
>
> I have never received such email.

Maybe sended only to domain register administrator  ("Amazon" team) or hosting "super" administrator ("Amazon" team).

**Comment by okiehsch** [comment]
I restored all blocks for the .xyz domains and removed the rest.
The .xyz filters are all valid badware entries.

**Comment by randomtroller** [comment]
Ok so I got some domains wrong here but why did you remove all of them? Most of them are valid badware entries.

**Comment by uBlock-user** [comment]
Got reported twice because you didn't do your due diligence and had to disable that to avoid any further breakage/blockage. If you're reporting domains, do a complete individual domain check.

**Comment by randomtroller** [comment]
> > This note is a false claim:
> > > I've also emailed to abuse email of the domain registrar but got no response.
> > 
> > 
> > I have never received such email.
> 
> Maybe sended only to domain register administrator ("Amazon" team) or hosting "super" administrator ("Amazon" team).



> Why is studeersnel.nl on this list?
> 
> I'm the co-founder of https://www.studeersnel.nl, which is actually the Dutch version of studocu.com, and it is definitely legit.
> We are not infected, it's not malware, it's not spam and it's definitely not redirecting to Slovenia or Turkish websites.
> 
> This note is a false claim:
> 
> > I've also emailed to abuse email of the domain registrar but got no response.
> 
> I have never received such email.
> 
> Could you please remove my domain since it's now blocking my users, who also use uBlock, and completely breaking the website.
> 
> Note:
> This list seems completely random and there are a lot of very genuine and legitimate websites on it. I would recommend removing this list and reviewing all items one by one.

I checked the domains to verify your claim and only found 4 that were legit websites. I don't know you but for me that's not "a lot".

**Comment by Kuijkens** [comment]
> I checked the domains to verify your claim and only found 4 that were legit websites. I don't know you but for me that's not "a lot".

This is a list that will cause websites to become unusable and break when using uBlock. It should be composed with the uttermost precision to not cause harm to legitimate businesses and as a side effect harm the reputation of uBlock. You put websites like mine (studeersnel.nl) and uva.nl (website of the university of Amsterdam) on it, both of them are well known and respected websites. 

1 = perhaps a mistake
2 = a pattern
3+ is a lot and makes your list questionable to say the least.

What's even more frustrating is that you claim to have (tried to) contact the domain registrar. I'm, besides the co-founder, the Head of DevOps for StudeerSnel and have access to all e-mail addresses that you could have possibly used to contact me (support/registrar/admin/administrator/lucas/sander at studeersnel.nl).
You didn't contact me.

> Maybe sended only to domain register administrator ("Amazon" team) or hosting "super" administrator ("Amazon" team).

If you contacted AWS instead, which is the DNS service provider, I would suggest to re-check your entire list and try to contact the Registrar instead.

**Comment by okiehsch** [comment]
> Ok so I got some domains wrong here but why did you remove all of them? Most of them are valid badware entries.

The domains I removed are all only used for redirection purposes, they are no landing pages as far as I can see.
We are not a dedicated malware filter project and simply lack the resources to test in detail a report of nearly 100 domains that may be malicious or not.
Best practice is to report such a huge list to a dedicated anti malware site.

@Kuijkens sorry for the trouble we caused,
